content/tutorials/drift-detection/index.md

```diff
@@ -0,0 +1,198 @@
+---
+title_tag: "Manage Infrastructure Drift with Pulumi Deployments | Tutorials"
```

Comment:
Should the title even mention Pulumi Deployments? It should be more about the use case of what the user wants to accomplish rather than how to use product X.  Remember https://developers.cloudflare.com/style-guide/documentation-content-strategy/content-types/tutorial/

I think that is true with https://www.pulumi.com/tutorials/pulumi-deployments-click-to-deploy/ this might need to get renamed because its not use case focused. Its actually just a how-to for a product. Like the title probably should be "Set up CICD Automation for Infrastructure" or something like that. Then there is probably a tutorial that shows the 3rd party way to do it by wiring it to their existing CICD solution.

Reply:
thx Aaron; I've adjusted the title.